src/_includes/blocks/footer.njk

```diff
@@ -1,11 +1,81 @@
- <footer class="border-t text-xs pt-4 mt-4">
+ <div class="text-xs mt-4 mx-4 mb-0">
```

Comment:
Can we bump the font size up across the board in the footer; `xs` -> `s`, `s` -> base?